OSDOCS-3704: Adding release notes for secondary scheduler operator 1.0.1 (#46227)
openshift/openshift-docs: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc

For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].

[id="secondary-scheduler-operator-release-notes-1.0.1"]
== Release notes for {secondary-scheduler-operator-full} 1.0.1

Issued: 2022-07-28

The following advisory is available for the {secondary-scheduler-operator-full} 1.0.1:

* link:https://access.redhat.com/errata/RHSA-2022:5699[RHSA-2022:5699]

[IMPORTANT]
====
If you are using {secondary-scheduler-operator-full} 1.0.0, it is expected to be automatically upgraded to version 1.0.1 when you update to the next minor release of {product-title}.
====

[id="secondary-scheduler-operator-1.0.1-new-features-and-enhancements"]
=== New features and enhancements

* The maximum {product-title} version for {secondary-scheduler-operator-full} 1.0.1 is 4.11.

[id="secondary-scheduler-1.0.1-bug-fixes"]
=== Bug fixes

> * Previously, the secondary scheduler deployment was not deleted after the secondary scheduler custom resource (CR) was deleted, which caused the {secondary-scheduler-operator} and operand not to be fully uninstalled. The secondary scheduler deployment is now deleted when the secondary scheduler CR is deleted, so that the {secondary-scheduler-operator} can now be fully uninstalled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100923[*BZ#2100923*])

Reviewer: The double negative here is a little hard to parse, so suggestion to change sentence to something like the following:
```suggestion
* Previously, the secondary scheduler deployment was not deleted after the secondary scheduler custom resource (CR) was deleted, which prevented the {secondary-scheduler-operator} and operand from being fully uninstalled. The secondary scheduler deployment is now deleted when the secondary scheduler CR is deleted, so that the {secondary-scheduler-operator} can now be fully uninstalled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100923[*BZ#2100923*])
```

Author: Great suggestion! I wasn't a fan of that wording either, but couldn't think of anything better :)

> * Previously, the secondary scheduler deployment was not deleted after the secondary scheduler custom resource (CR) was deleted, which prevented the {secondary-scheduler-operator} and operand from being fully uninstalled. The secondary scheduler deployment is now deleted when the secondary scheduler CR is deleted, so that the {secondary-scheduler-operator} can now be fully uninstalled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100923[*BZ#2100923*])

Reviewer 2: @bergerhoffer I see this is a  4.11 bug here which is fixed in SSO 1.1.0, so i do not think it should be listed here.

Author: Confirmed via slack that this BZ was a bug in 1.0 and fixed in 1.0.1, so is appropriate to list here
